stylesheets/_layout.tabs.scss

```diff
@@ -232,3 +232,8 @@
         @include col-span-width(9);
     }
 }
+
+.tab__header {
+    flex-basis: 100%;
```

Comment:
Feel like this should be set within `_layout.settings.scss` as that's the only place `.tab__header` would be within an element with `display: flex` (other layouts don't use flex), perhaps nested within the the rule block that contains the `flex-wrap` change above.